src/signals/incident/services/convert-value/index.ts

```diff
@@ -0,0 +1,27 @@
+import { isArray } from 'utils';
+
+const convertValue = (value: unknown = undefined, postPropertyNames: string[] = []): any => {
+  if (value === 0) {
+    return 0;
+  }
+
+  if (value === true) {
+    return 'ja';
+  }
+
+  if (value === false) {
+    return 'nee';
+  }
+
+  // when property names are specified, create an array with objects with only those properties
+  if (isArray(value) && postPropertyNames.length) {
```

Comment:
Can't we use `Array.isArray`? That would also remove the need to cast to an array below

Reply:
It's preferred to use `unknown` instead of `any`